docs/Concepts/Node-Keys.md

```diff
@@ -42,6 +42,13 @@ You can export the node address, either to standard output or to a specified fil
 The enode URL identifies a node. For example, the `--bootnodes` option and the
 `perm_addNodesToAllowlist` method specify nodes by enode URL.
 
+!!! tip
+
+    If deploying Besu using Kubernetes in private permissioned network, use the
+    [`--Xdns-enabled`](../Reference/CLI/CLI-Syntax.md#xdns-enabled) option to use domain names
+    instead of IP addresses. This ensures that Besu can connect to a container even if the IP
```

Comment:
I would say that this option alone does not do that. With this option, Besu will query the DNS when it starts up and if the container IP changes afterwards it will not detect it. You must add the second option `Xdns-update-enabled` so that Besu query the DNS each time and detect a change of IP

Reply:
Done.